index.html

```diff
@@ -2374,12 +2374,14 @@ <h5><code>subscribeevent</code></h5>
             <code>EventAffordance</code> and MUST be serialized in JSON.
             The <code>id</code> field SHOULD be set to a unique identifier for 
             the event, for use when re-establishing a dropped connection (see 
-            below).
+            below). It is RECOMMENDED that the identifier is a timestamp 
```

Comment:
Recommending ISO 8601 is not enough  to guarantee interop.

Reply:
I think we should define a structured event payload that contains the timestamp and id as members. 
We will do other event bindings, and this protocol specific encoding will prevent interop.

Reply:
@mlagally wrote:
> Recommending ISO 8601 is not enough to guarantee interop.

Can you explain why? I have a feeling that the potential ambiguities you are concerned about (e.g. 24 vs. 00 and the Z separator) are fixed in the latest version.

> I think we should define a structured event payload that contains the timestamp and id as members.
We will do other event bindings, and this protocol specific encoding will prevent interop.

Again, protocol bindings are meant to be protocol specific, that's the whole point. See https://github.com/w3c/wot-profile/issues/126#issuecomment-971507879

Reply:
Potential alternative reference: [RFC3339](https://www.rfc-editor.org/rfc/rfc3339)